packages/extension-record-video/src/index.ts

```diff
@@ -58,10 +58,18 @@ class RecordVideoExtension implements JsPsychExtension {
 
       if (!this.currentTrialData.record_video_data) {
         this.onUpdateCallback = () => {
-          resolve(this.currentTrialData);
+          resolve({
```

Comment:
Hmm - shouldn't we be able to add this in the core library so each extension doesn't need to do it manually?

Reply:
Yes! This has been changed.